libs/wingsdk/src/std/resource.ts

```diff
@@ -80,6 +80,14 @@ export interface ILiftable {
  * to access the lifted object inflight.
  */
 export interface IHostedLiftable extends ILiftable {
+  /**
+   * Compiler-generated data that describes the dependencies of this object on other
+   * objects. This is used to determine which permissions need to be granted to the
+   * inflight host.
+   * @internal
+   */
+  _onLiftDeps?: LiftDepsMatrixRaw;
```

Comment:
Wasn't sure what to name this internal property. I guess since it's internal it doesn't matter too much, but suggestions are still welcome